src/server/managers/registryManager.js

```diff
@@ -13,6 +13,20 @@ class RegistryManager extends BaseManager {
         return Registry.create(obj);
     }
 
+	findByInstanceId(instanceId) {
```

Comment:
unused method?

Reply:
I don't now))

Reply:
so findByInstanceId isn't used, remove it